semantic/src/site/globals/site.overrides

```diff
@@ -548,6 +548,7 @@ a:hover {
     gap: 10px;
     max-width: 650px;
     margin-top: 20px;
+    margin-bottom: 60px; // to offset the Actions area that covers it.
```

Comment:
This is to make the scrolling work properly for Preview modal. I thought I'd already fixed this, but I hadn't allowed for the height of the "Actions" area at the bottom (with the OK button), which covers the main content.

The actions area is absolutely positioned, so the main content sits behind it and doesn't show a scroll bar until it's taller than the whole preview area, including the area covered by the actions area. This extra margin is the height of the actions area, so content won't get stuck behind it without a scroll bar.